website/app/templates/head.hbs

```diff
@@ -21,45 +21,53 @@ We've made some small customizations to simplify this compared to the default te
 
 {{! END GENERAL }}
 
-{{! BEGIN OPENGRAPH }}
+{{! BEGIN TWITTER }}
 
-{{#if this.model.siteName}}
-  <meta property="og:site_name" content={{this.model.siteName}} />
+<meta name="twitter:card" content="summary_large_image" />
+
+{{#if this.model.twitterUsername}}
+  <meta name="twitter:site" content={{this.model.twitterUsername}} />
+  {{!-- <meta name="twitter:creator" content={{this.model.twitterUsername}} /> --}}
 {{/if}}
 
 {{#if this.model.title}}
-  <meta property="og:title" content={{this.model.title}} />
+  <meta name="twitter:title" content={{this.model.title}} />
 {{/if}}
 
 {{#if this.model.description}}
-  <meta property="og:description" content={{this.model.description}} />
+  <meta name="twitter:description" content={{this.model.description}} />
 {{/if}}
 
 {{#if this.model.imgSrc}}
-  <meta property="og:image" content={{this.model.imgSrc}} />
-  <meta property="og:image:type" content="image/jpg" />
+  <meta name="twitter:image" content={{this.model.imgSrc}} />
 {{/if}}
 
-{{! END OPENGRAPH }}
+{{!-- {{#if this.model.url}}
+  <meta name="twitter:url" content={{this.model.url}} />
+{{/if}} --}}
 
-{{! BEGIN TWITTER }}
+{{! END TWITTER }}
 
-<meta name="twitter:card" content="summary" />
+{{! BEGIN OPENGRAPH }}
 
-{{#if this.model.imgSrc}}
-  <meta name="twitter:image" content={{this.model.imgSrc}} />
-{{/if}}
+<meta property="og:url" content="https://hds-website-git-hds-1383-twitter-preview-hashicorp.vercel.app/" />
```

Comment:
I would move the actual url to `url` in the config file. I think for our short term purposes we could hardcode it to https://hds-website.vercel.app/ but there is likely a way we could have it be dynamic as well.

Reply:
Pushed update to use the variable. Let me know if I didn't use the right thing.